server/src/main/java/org/elasticsearch/action/admin/cluster/reroute/ClusterRerouteRequest.java

```diff
@@ -34,8 +35,8 @@ public ClusterRerouteRequest(StreamInput in) throws IOException {
         retryFailed = in.readBoolean();
     }
 
-    public ClusterRerouteRequest() {
-        super(TRAPPY_IMPLICIT_DEFAULT_MASTER_NODE_TIMEOUT, DEFAULT_ACK_TIMEOUT);
```

Comment:
The objective of this PR (and many like it) is to remove one or more usages of `TRAPPY_IMPLICIT_DEFAULT_MASTER_NODE_TIMEOUT`. Here this is the one that's going away.